core/src/executor/evaluate/function.rs

```diff
@@ -50,6 +50,22 @@ macro_rules! eval_to_float {
 
 // --- text ---
 
+pub fn concat(name: String, exprs: Vec<Evaluated<'_>>) -> Result<Value> {
+    let mut s = String::from("");
+
+    for expr in exprs {
+        match expr.try_into()? {
+            Value::Str(value) => s.push_str(&value),
+            Value::Null => return Ok(Value::Null),
+            _ => {
+                return Err(EvaluateError::FunctionRequiresStringValue(name).into());
+            }
+        }
+    }
+
+    Ok(Value::Str(s))
+}
```

Comment:
`Value` itself has `concat` method using `||` binary operator.
https://github.com/gluesql/gluesql/blob/1213d8288a427745862c0b1c40f83fa057033fd3/core/src/data/value/mod.rs#L165-L170

ref. [test-suite/ concat.rs](https://github.com/gluesql/gluesql/blob/main/test-suite/src/concat.rs)

If you don't have intend `||` and `CONCAT` to have different behaviors, then how about to use `Value::concat`?

Reply:
concat has a variable number of arguments, (not just  2).

Reply:
I think the concat function should have a different behavior (ie, ignore nulls), but it was suggested that I create another issue so we can discuss this.